diesel_cli/src/migrations/mod.rs

```diff
@@ -93,7 +93,16 @@ pub(super) fn run_migration_command(matches: &ArgMatches) -> Result<(), crate::e
             {
                 let config = Config::read(matches)?;
                 let diff_schema = if diff_schema == "NOT_SET" {
-                    config.print_schema.file.clone()
+                    if config.print_schema.all_configs.len() != 1 {
+                        return Err(crate::errors::Error::UnsupportedFeature(
```

Comment:
This should default to the default schema if no other schema key is set explicity